core/lib/json_api_client_extension/json_api_client_orm_adapter.rb

```diff
@@ -15,14 +15,14 @@ def column_names
 
     # @see OrmAdapter::Base#get!
     def get!(id)
-      res = klass.includes(*klass::INCLUDED_DEPENDENCIES).find(wrap_key(id)).first
+      res = klass.find(wrap_key(id)).first
```

Comment:
Are you removing the `INCLUDED_DEPENDENCIES` feature? Was it widely used or just for the current_user?

Reply:
it was mainly used for the current_user, but also for other controllers. (for example when `current_user.organizations` was called)